modules/cco-ccoctl-configuring.adoc

```diff
@@ -138,19 +172,24 @@ Available Commands:
   gcp          Manage credentials objects for Google cloud
   help         Help about any command
   ibmcloud     Manage credentials objects for IBM Cloud
+  powervs      Manage credentials objects for IBM Power VS
```

Comment:
there's not powervs option. if you see https://github.com/openshift/cloud-credential-operator/blob/master/docs/ccoctl.md#ibm-cloud-power-vs you can see that we use the ibmcloud one (since we both use IBM Cloud acounts)